account_invoice_fixed_discount/models/account_move_line.py

```diff
@@ -73,7 +73,7 @@ def _onchange_discount(self):
         """
         if self.env.context.get("ignore_discount_onchange"):
             return
-        self.env.context = self.with_context(ignore_discount_onchange=True).env.context
+        self = self.with_context(ignore_discount_onchange=True)
```

Comment:
@yankinmax And why not moving `discount_fixed` to computed/stored/readonly=False field instead ?